cms_index.php

```diff
@@ -28,19 +49,32 @@
         <textarea class="typeText" name="addAboutMeText" form="addAboutMeText"></textarea>
         <input type="submit" value="Add text">
     </form>
+    <?php
+    echo $addAboutMeSuccess;
+    ?>
     <form method="post">
-        <label for="selectAboutMeText"><h4>Edit text:</h4></label>
+        <label for="selectAboutMeText"><h4>Edit text or quote:</h4></label>
         <select class="aboutMeDropdown" name="selectAboutMeText">
-
+            <?php
+            echo $displayEditAboutMeDropdown;
+            ?>
         </select>
         <input type="submit" value="select text">
     </form>
-    <form method="post">
-            <textarea class="typeText" name="editAboutMeText" form="editAboutMeText">
-
+    <form method="post" action="" id="editAboutMeTextAndQuote">
+            <textarea class="typeText" name="editAboutMeTextAndQuote" form="editAboutMeTextAndQuote">
+<?php echo $displayAboutTextToEdit;?>
             </textarea>
-        <input type="submit" value="Edit text">
+                <?php
+                if (isset($editId)) {
+                    echo '<input type="hidden" value=' . $editId . ' name="editId">';
+                    }
+                echo $displaySubmitEditButton;
+                ?>
     </form>
+    <?php
+    echo $editAboutMeSuccess;
```

Comment:
put an isset() like charlie said for aldos code review

Reply:
Removed since I had to chose between this and the page reload